program-runtime/src/invoke_context.rs

```diff
@@ -229,6 +231,8 @@ impl<'a> InvokeContext<'a> {
             timings: ExecuteDetailsTimings::default(),
             syscall_context: Vec::new(),
             traces: Vec::new(),
+            #[cfg(feature = "stub-proc-instr")]
+            proc_instr_callback: None,
```

Comment:
Do you mind if we change this to be non-optional?

Reply:
discussed offline: agreed to switch this to non-optional to reduce complexity of fuzz harnesses

Reply:
discussed offline: we are not doing this